src/core.test.ts.md

```diff
@@ -282,4 +282,16 @@ Generated by [AVA](https://avajs.dev).
             return var8[arg0][arg1];␊
         }␊
     }␊
+    ␊
+    contract $WithInternalReturns is WithInternalReturns {␊
+        event $incrementInternalReturn(uint256 arg0);␊
```

Comment:
Would name the events `*Returned`.

Reply:
this was not updated. The name are way longer (using fn args types to avoid clash). will have these end with `Returned`